otx/algorithms/action/adapters/mmaction/models/backbones/movinet.py

```diff
@@ -0,0 +1,1048 @@
+"""
```

Comment:
In my opinion, it seems that not only movinet but also classes that can be general are mixed. The file is too long to look appropriate. This looks like it needs an update.

Reply:
We agree on that, however it should be done after action task support basic train, eval, openvino task and test
Currently our focus is enabling feature first, the general refactoring will be done by this year
However, if you think the change is needed urgently, then we should think about it

Reply:
I think this should be addressed in this PR. There seems no reason to delay this.

Reply:
@JihwanEom Let's clean implementation of movinet structure. @harimkang and I can discuss of direction of refactoring